library/VTKExtensions/Rendering/vtkF3DInteractorStyle.cxx

```diff
@@ -141,7 +141,109 @@ void vtkF3DInteractorStyle::Dolly(double factor)
   {
     return;
   }
-  this->Superclass::Dolly(factor);
+  if (this->Interactor->GetControlKey())
+  {
+    vtkF3DInteractorStyle::DollyToPosition(
+      factor, this->Interactor->GetEventPosition(), this->CurrentRenderer);
+  }
+  else
+  {
+    this->Superclass::Dolly(factor);
+  }
+}
+
+//----------------------------------------------------------------------------
+void vtkF3DInteractorStyle::DollyToPosition(double factor, int* position, vtkRenderer* renderer)
+{
+  if (this->CameraMovementDisabled)
+  {
+    return;
+  }
+  vtkCamera* cam = renderer->GetActiveCamera();
+  if (cam->GetParallelProjection())
```

Comment:
we do not support parallel project yet I think

#17 #1096 

There is a PR adding it but its not merged yet #1098

Reply:
Should remove the parallel projection part, or just let it be so that it can be used when parallel projection is added later

Reply:
Lets remove it for now.